Sources/Kafka/KafkaConsumer.swift

```diff
@@ -367,17 +369,46 @@ public final class KafkaConsumer: Sendable, Service {
     /// - Parameters:
     ///     - message: Last received message that shall be marked as read.
     /// - Throws: A ``KafkaError`` if committing failed.
+    public func triggerCommit(_ message: KafkaConsumerMessage) throws {
```

Comment:
Okay sorry for the bike shedding but we probably should call this `scheduleCommit` which has a bit better semantics. Can you rename the public and internal methods?